lib/bump/tasks.rb

```diff
@@ -25,6 +25,7 @@
       end
       options = {
         tag: ENV['TAG'],
+        tag_prefix: ENV['TAG_PREFIX'],#tag_prefix_value,
```

Comment:
```suggestion
        tag_prefix: ENV['TAG_PREFIX'],
```

Reply:
... we should not repeat the default, do not set the option if env var was not set